app/models/course/assessment/answer.rb

```diff
@@ -2,7 +2,7 @@
 class Course::Assessment::Answer < ActiveRecord::Base
   include Workflow
   actable
-  acts_as :discussion_topic, class_name: Course::Discussion::Topic.name
+  acts_as_discussion_topic display_global: true
```

Comment:
I'm a bit concerned global might be some reserved keyword somewhere. Also that "global" itself doesn't really say what it does. Comments are OK but I prefer an obvious name.